sdk/deviceupdate/Azure.Iot.DeviceUpdate/src/Azure.IoT.DeviceUpdate.csproj

```diff
@@ -9,6 +9,8 @@
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <!-- These supressions should be removed in a production library -->
     <NoWarn>$(NoWarn);CS1591;AZC0001;AZC0012</NoWarn>
+    <AssemblyName>Azure.IoT.DeviceUpdate</AssemblyName>
```

Comment:
I believe these are redundant as they default from the project which you renamed it these shouldn't be needed.

Reply:
Sounds good removed.